.github/workflows/push-validation.yml

```diff
@@ -15,4 +15,6 @@ jobs:
         java-version: 14
         java-package: jdk
     - name: Build and test with Gradle
+      env:
+        JAVA_TOOL_OPTIONS: -Dtestfx.robot=glass -Dtestfx.headless=true -Dprism.verbose=true
```

Comment:
Should a developer use these flags locally as well?

Reply:
Only if you want to make sure your test environment matches the GitHub one. And there's certainly some value to that if you're trying to reproduce a failure that GitHub tells you about!

but it's not necessary if your development environment is running somewhere it can just as easily use a standard JavaFX backend.